packages/core/src/feed.ts

```diff
@@ -29,11 +30,43 @@ class DocumentsSubject extends Subject<FeedDocument> {}
 export interface PublicFeed {
   aggregation: {
     documents: Observable<FeedDocument>
+    documentsA: (gt?: string) => ReadableStream<FeedDocument>
```

Comment:
Can we have a more descriptive name for this? Maybe `readableDoc` or something like that, or document it inline?

Reply:
Yeah, we do not need previous `documents` based on `Observable`. Deleted that. Renamed `documentsA` to `documents`.